### PR TITLE
2021.01.1

### PR DESCRIPTION
**Pull request by oblandford-novetta**
Fixes # .

Changes offered/proposed in this pull request:
- 
- 
- 

* New PR Alert to: @plus3it/spel

### CODE REVIEW

**Review by lorengordon — CHANGES_REQUESTED** [review]
One small change. And need to run `bumpverson major` to roll over the year....

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 2021.01.1
+
+**Released**: 2020.01.26
+
+**Commit Delta**: [Change from 2020.12.1 release](https://github.com/plus3it/spel/compare/2020.12.1...2021.01.1)
```
Comment (lorengordon) [inline CHANGELOG.md:7]:
```suggestion
**Commit Delta**: [Change from 2021.01.1 release](https://github.com/plus3it/spel/compare/2020.12.1...2021.01.1)
```

Comment (eemperor) [reply CHANGELOG.md:7]: This looks okay.  It's the manifests url that needs to be updated.

**Review by eemperor — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (lorengordon) [reply CHANGELOG.md:7]: Oops @eemperor is right. I meant to comment on the other line in my review, but think i forgot to "start a review", then submitted the request for change, then realized my comment was missing, then attempted to add the comment i meant to make, and apparently made it on the wrong line. So here we are.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 2021.01.1
+
+**Released**: 2020.01.26
+
+**Commit Delta**: [Change from 2021.01.1 release](https://github.com/plus3it/spel/compare/2020.12.1...2021.01.1)
```
Comment (lorengordon) [inline CHANGELOG.md:7]: @oblandford-novetta sorry this was my bad, the "change from" text was correct before. i meant to comment on the "manifests" link.

```suggestion
**Commit Delta**: [Change from 2020.12.1 release](https://github.com/plus3it/spel/compare/2020.12.1...2021.01.1)
```

Comment (oblandford-novetta) [reply CHANGELOG.md]: OK. That is my bad as well. I should have been paying more attention when making the suggested update. I was just like a robot. I also should have done all of this with a find replace in the first place.

**Review by eemperor — APPROVED** [review]
Looks good to me

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+
+**Manifests**: <https://github.com/plus3it/spel/blob/2021.01.1/manifests>
+
+**Summary**:
+
+*   "Extra" packages updated in this release:
+    - aws-cli/1.18.219
+    - aws-cli/2.1.21
+*   RHEL-only:
+    - rh-amazon-rhui-client-3.0.39
```
Comment (lorengordon) [inline CHANGELOG.md:17]: @eemperor i wonder if this client version indicates this is RHUIv3?

Comment (eemperor) [reply CHANGELOG.md:17]: Yeah.  Might be.

**Comment by oblandford-novetta** [comment]
The bumpversion seems right? Can someone confirm this and if this is ready to merge?

**Comment by eemperor** [comment]
@lorengordon any additional changes?

**Comment by lorengordon** [comment]
No it's good

**Comment by eemperor** [comment]
@lorengordon seems still stuck on your previous requested changes.

**Comment by lorengordon** [comment]
Oh, you can "dismiss review" when that happens. :)

**Review by lorengordon — APPROVED** [review]

**Comment by eemperor** [comment]
Ah!  Got it.

**Pull request merged**